bindings.c

```diff
@@ -1155,6 +1156,7 @@ bool cgfs_get_value(const char *controller, const char *cgroup, const char *file
 	char *fnam, *tmpc;
 
 	tmpc = find_mounted_controller(controller, &cfd);
+	lxcfs_debug("tmpc: %s\n", tmpc);
 	if (!tmpc)
 		return false;
```

Comment:
He, this would be a NULL-dereference if `find_mounted_controller()` fails, no? :)